SQL.js

```diff
@@ -44,25 +47,40 @@ class SqlStatement {
 
   generateString (type) {
     let text = this.strings[0]
+    let valueOffset = 0
+    const values = [...this._values]
 
     for (let i = 1; i < this.strings.length; i++) {
-      let delimiter = '?'
-      if (type === 'pg') {
-        delimiter = '$' + i
+      if (values[i - 1 + valueOffset] && values[i - 1 + valueOffset][unsafe]) {
+        text += `${QUOTE}${values[i - 1 + valueOffset].value}${QUOTE}${
+          this.strings[i]
+        }`
```

Comment:
Why are we adding quotes to unsafe values?

I would have two different functions:
- `unsafe` that just mark input to be inserted without changes
- `quoteIdent` that add the quotes 

`quoteIdent` may be something like:
```js
module.exports.quoteIdent = value => unsafe(`${QUOTE}${value}${QUOTE}`)
```

Otherwise I'd find the name `unsafe` misleading.

Reply:
that's a good point. if we do that, that would raise a question about whether `append(..., { unsafe: true })` has a reason to stay, but I guess we can keep both, at the cost of making the API a little more difficult to understand.

Reply:
Yes, that's a little deviation from current api. 
If that's a problem, we can decide not to expose `unsafe` and have it only used internally.

Reply:
@paolochiodi I added quoteIdent based on your suggestion and also exposed unsafe as a function which is an alternative way to using the existing unsafe option